src/humanPose/HumanPoseAnalyzerSettingsUi.js

```diff
@@ -112,6 +112,16 @@ export class HumanPoseAnalyzerSettingsUi {
     }
     
     setUpEventListeners() {
+        this.root.addEventListener('pointerdown', (event) => {
+            event.stopPropagation();
+        });
+        this.root.addEventListener('pointermove', (event) => {
+            event.stopPropagation();
```

Comment:
This blanket stopPropagation has the unfortunate effect that if you start a camera interaction (e.g. right click to pivot) then go over the settings ui it first stops rotating due to losing pointermove then doesn't stop the camera interaction since pointerup also gets stopped. I think the more durable solution is to check event.target in the spaghetti's pointer listeners. This would also fix clicking on other places like the top menu bar resetting the spaghetti region